src/DataBase/PiggyDbContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ePiggyWeb.DataBase.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace ePiggyWeb.DataBase
+{
+    public class PiggyDbContext : DbContext
+    {
+        public DbSet<UserModel> Users { get; set; }
+        public DbSet<ExpenseModel> Expenses { get; set; }
+        public DbSet<IncomeModel> Incomes { get; set; }
+        public DbSet<GoalModel> Goals { get; set; }
+        public PiggyDbContext(DbContextOptions<PiggyDbContext> options) : base(options) {}
+
+    }
+}
```

Comment:
Was this file just renamed, or new one created?

Reply:
This is a new one, I've left the old Database Context intact for now.

Reply:
Granted it's almost the same as old Database context, with minor differences in it's creation